SU-SWS/linky_clicky
# External Link Module Behat test

#  READY FOR REVIEW

# Summary
These are behat tests for the external link module

# Needed By (Date)
- Soon. It's needed to put the External Link module on SRWC

# Urgency
- Important for us to go forward putting the External Link module on SRWC

# Steps to Test

I added a "Local" site to make it easier to test. Let me know if I should take it out.

1. Navigate to `linky_clicky/sites/local`
2. Create your behat.local.yml file
3. Run the External Link behat tests
`../../bin/behat -s dev -p local-local features/drupal.org/extlink/extlink.feature`

# Affected Projects or Products
- Needed for the SRWC site

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/LBRESRWC-74
- This may be a mistake. These Behat tests rely on the _Stanford Landing Page_ module. This was because I that's how they want to use it on the SRWC site. 
- Anyone who should be notified? (`@mention` them here)
@alisonho 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
